src/content/docs/en/anywhere/rpi.mdx

```diff
@@ -7,58 +7,43 @@ import Alert from "../../../../components/Docs/Alert.astro";
 
 Raspberry Pi is a very popular series of SBCs. They are cheap, easy to get, run Linux well, and have a large ecosystem of expansion cards (Hats and Bonnets) and accessories.
 
-Fedora has upstream support for the Pi 4, 400, and 3. Ultramarine Anywhere intends to extend support to the Zero 2 and 5.
+Raspberry Pi support in Ultramarine Linux is provided in partnership with the Raspberry Pi Foundation.
 
 ## Raspberry Pi 5
 
 <Alert type="info">
-  This device type is still in the **Planning Phase**. All information on this
+  This device type is still in the **Testing**. All information on this
   page is subject to change.
 </Alert>
 
 The Raspberry Pi 5 is not currently supported in Fedora. We intend to package the official Raspberry Pi kernel (which should also provide a better experience on the other Pi models.)
 
 ## Raspberry Pi 4 and 400
 
-The Raspberry Pi 4 and 400 have had support in Ultramarine Linux since UM39, and desktop images since UM40. We use the Fedora kernels and boot proccess.
+The Raspberry Pi 4 and 400 have had support in Ultramarine Linux since UM39, and desktop images since UM40. We use the Raspberry Pi Foundation's kernel and Fedora's boot proccess.
 
 ### Notes
 
-- RPi4B has a 1GB and 2GB model, we suggest against running a desktop environment on these (Xfce may be alright.)
-- RPi400 does not have support for WiFi in Ultramarine, this is an upstream issue.
+- RPi4B has a 1GB and 2GB model, we suggest you don't run a desktop environment on these (Xfce may be alright.)
 - The Pi 4 series has native USB boot support. Your experience may be better with a fast USB Drive or SSD connected through an adapter. See [this article](https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#usb-mass-storage-boot) for more info.
 - If using a microSD card, we reccomend using a Class 10 microSD card, your experience will be signifcantly better than lower classes.
 - If using a USB drive, ensure it's USB 3.0 or newer, 2.0 will be a signifcantly worse experience than a microSD.
-- Your storage device will need to be larger than 9GB (the image is 8.59GB) We suggest 16GB so you have space for software.
-
-### Installing
-
-1. Install [Raspberry Pi Imager](https://flathub.org/apps/org.raspberrypi.rpi-imager).
-2. Simply select "Other General Purpose OS" then your Ultramarine Edition of choice.
-3. Insert the storage device into your Pi and boot up! The root filesystem will automatically be expanded.
-4. You will be prompted to set up a user, do this and you're done!.
+- Your storage device will need to be larger than 16GB (the image is ~15GB) We suggest 20GB so you have space for software.
```

Comment:
Raw images consists of mostly blank space:
![image](https://github.com/user-attachments/assets/a4a6f63d-ab77-45a6-ae52-d87c36a822d5)

Maximum is around 7.1GB (plasma), final raw image size is 12.1 GB, 3.6 GB after compression, nowhere to 15GB.

If I understand this correctly, a system with 10 GB can install plasma with 3 GB to spare. This of course requires verifications.

If true, also edit rpi3 notes.

Reply:
compression doesn't matter here, we're decompressing, i swore they were like 12-15gb now, thanks for the correction

Reply:
The largest is 15GBs, the smallest is 8GBs. There is free space within those images but they still take up that much space once decompressed.

@madonuko didn't you have a way to shrink these sizes down? Especially the plasma one? If we can shrink down the plasma one, the next largest raw image is 12GBs.

Reply:
> The largest is 15GBs, the smallest is 8GBs. There is free space within those images but they still take up that much space once decompressed.
> 
> @madonuko didn't you have a way to shrink these sizes down? Especially the plasma one? If we can shrink down the plasma one, the next largest raw image is 12GBs.

![image](https://github.com/user-attachments/assets/2d314755-d199-48d8-ab7a-d7916fe6d1ad)

Reply:
You're showing imgs from 2 Aug, that's old as heck.

Reply:
Looks like the most recent KDE raw image is 13GBs.

Reply:
Wait that's x86, the ARM raw image is probably still 15, checking now.